eos/models/standard-model.hh

```diff
@@ -242,6 +242,21 @@ namespace eos
             virtual WilsonCoefficients<wc::DBCU> wet_dbcu(const bool & cp_conjugate) const;
     };
 
+    template <> class SMComponent<components::WET::SCNuL> :
+    public virtual ModelComponent<components::WET::SCNuL>
+    {
+        private:
+            UsedParameter _alpha_e__scnul;
+            UsedParameter _m_Z__scnul;
+            std::array<UsedParameter, 3u> _mu__scnul;
+
+        public:
+            SMComponent(const Parameters &, ParameterUser &);
+
+            /* b->c Wilson coefficients */
```

Comment:
Typo in comment: should be c->s I assume

Reply:
Thanks, fixed!